src/components/MediaCreationModal/MediaCreationModal.jsx

```diff
@@ -60,7 +60,7 @@ export const MediaCreationModal = ({
           id="file-upload"
           accept={
             mediaRoom === "images"
-              ? "image/jpeg, image/png, image/gif, image/svg+xml, image/tiff, image/bmp, image/x-icon"
+              ? "image/jpeg, image/png, image/gif, image/svg+xml, image/tiff, image/bmp, image/x-icon, image/webp"
```

Comment:
this is fine for now cos there's only 1 call site but perhaps we should shift this to a constant and place this in an appropriate file for easy discovery?

Reply:
Shifted out in https://github.com/isomerpages/isomercms-frontend/pull/1180/commits/1fbc5c845d558299b123ec0568659337e6a15d8f!